### PR TITLE
DOC-203 Update for S3-compatible storage

### PR DESCRIPTION
**Pull request by bmansheim**

### CODE REVIEW

File: content/embeds/backup-locations.md [hunk 1]
```diff
@@ -35,12 +35,14 @@ For example: `sftp://username:password@10.1.1.1/home/backups/`
...
 ### AWS S3
 
-Before you choose to backup to Amazon S3, make sure that you have:
+Before you choose to backup to Amazon AWS S3, make sure that you have:
 
 - Storage location path in the format: `s3://bucketname/path/`
 - Access key ID
 - Secret access key
 
+To connect to a locally-hosted AWS S3 location, run: `rladmin cluster config s3_url <hostname>`
```
Comment (Paz-Yanover) [inline content/embeds/backup-locations.md:44]: @guyco - What do you think?
Please refer to two issues:
1. The name “locally-hosted AWS S3 location” - is it accurate name?
2. Do the customer need to handle any certs in order for this command to work?

Comment (Paz-Yanover) [reply content/embeds/backup-locations.md:44]: @guyco-redis - Thanks.
@bmansheim - Can you please update the PR accordingly?

Comment (bmansheim) [reply content/embeds/backup-locations.md]: Done

File: content/embeds/backup-locations.md [hunk 2]
```diff
@@ -41,7 +41,9 @@ Before you choose to backup to Amazon AWS S3, make sure that you have:
 - Access key ID
 - Secret access key
 
-To connect to a locally-hosted AWS S3 location, run: `rladmin cluster config s3_url <hostname>`
+You can also connect to a storage service that uses the S3 protocol but is not hosted by Amazon AWS.
+The storage service must have a valid S3 certificate.
```
Comment (guyco-redis) [inline content/embeds/backup-locations.md:45]: @bmansheim  "a valid SSL certificate" instead of S3 certificate (my mistake in the initial comment)

Comment (bmansheim) [reply content/embeds/backup-locations.md]: Done

**Pull request merged**